server/views/pages/calendar.ejs

```diff
@@ -28,8 +28,8 @@
         </div>
 
```

Comment:
added some tabindex-es so each date number is visited followed by the contents div